### ISSUE TITLE
Events occasionally display in incorrect order

### ISSUE THREAD

**Issue by bridgetkromhout** [devopsdays/devopsdays-web#6539]
![image](https://cloud.githubusercontent.com/assets/2104453/22747066/8d5cbdd2-edeb-11e6-8c9a-75e9098cda8a.png)

The code in question that produced the above screenshot of two different sidebars is here: https://github.com/devopsdays/devopsdays-theme/blob/master/layouts/partials/future.html

We occasionally get a different sort order for `range` queries without there being a code change. (See also the note in https://github.com/devopsdays/devopsdays-theme/issues/159#issuecomment-277492678). Bears investigation.

**added by @mattstratton** - No reproduction steps.

**Comment by mattstratton** [comment]
Note - this also occurs on the "other events from x city" on the welcome template

**Comment by kuritz** [comment]
On our current page (https://devopsdays.org/events/2020-new-york-city/welcome/) we have ended up with an order like the below a few times:

```
Other New York City Events
2013   2016   2019   2015   2014   2018   2013   
```

In the few times it's happened so far it seems to go out of order when the site is being rebuilt with changes to our content, and fixed when built for other reasons.

**Comment by mattstratton** [comment]
> I am wondering if we would do better to use the map approach used in `past.html`; that gives us a sorted map.

I really think this is probably the move.

I also have been wondering if we call the sidebar with `partialCached` at least it will be consistent on every page.

**Comment by mattstratton** [comment]
Fixed in #11923
